packages/regl-worldview/src/stories/GLText.stories.js

```diff
@@ -91,6 +170,30 @@ storiesOf("Worldview/GLText", module)
     }
     return <Example />;
   })
+  .add("changing text hires", () => {
+    function Example() {
+      const [text, setText] = useState("Hello\nWorldview");
+      useLayoutEffect(() => {
+        let i = 0;
+        const id = setInterval(() => {
+          setText(`New text! ${++i}`);
+          if (inScreenshotTests()) {
+            clearInterval(id);
+          }
+        }, 100);
+        return () => clearInterval(id);
```

Comment:
We automatically take screenshots of these stories in CI, after 200ms, which we use to catch regressions; see https://github.com/cruise-automation/webviz/pull/349/files#diff-c846e28069eda40e4003933c8306d5ffR47. So this would lead to a flakey screenshot test.

It might be better to just add a button or text field or so to change the text. It would still be nice to do one text change automatically though, so that the screenshot test also captures that changing text works properly.

Reply:
@janpaul123 I copy this story from the one above (see "changing text") and just added the "hires" option. I'm not sure I'm understanding what should be the correct approach. Should I update the behavior for "changing text" story as well?

Reply:
Ah yes, that one is bad too, would be good to fix that one as well.

Btw, you might have noticed that CI is a bit broken in that it often times out, but we have plans on fixing this soon-ish (we're considering using a different CI system so we're waiting on that decision).

Reply:
@janpaul123 @troygibb I updated the changingText story to make it more predictable. It changes the text once when initializing. There's also a button to change the text manually when using storyboard